openshift/buildConfig-template.yaml

```diff
@@ -923,3 +923,76 @@ objects:
           generic:
             secretReference:
               name: generic-webhook-secret
+  - kind: BuildConfig
+    apiVersion: v1
+    metadata:
+      name: "solver-rhel-8.2-py38"
```

Comment:
The minor version looks redundant here (also in other build configs already merged in this repo).